ops/cmd/sync_staging/main.go

```diff
@@ -57,8 +57,38 @@ func action(cliCtx *cli.Context) error {
 	}
 
 	stagingDir := paths.StagingDir(wd)
+	stagedSuperchainDefinition, err := manage.StagedSuperchainDefinition(wd)
+
+	l1RpcUrl := ""
+	if err == nil {
+		output.WriteOK("superchain definition found, finding L1 RPC URL...")
+		l1RpcUrl, err = config.FindValidL1URL(cliCtx.Context,
+			log.NewLogger(log.NewTerminalHandlerWithLevel(os.Stderr, log.LevelInfo, false)),
```

Comment:
You're creating two separate loggers. One here and one down in the codegen block of code. Might be better to just create one at the beginning of this action function and use it in both places

Reply:
[42d9ab6](https://github.com/ethereum-optimism/superchain-registry/pull/1011/commits/42d9ab669b96dbb402697a2e4e839cc5675d8425)